tests/test_actions.py

```diff
@@ -384,6 +384,7 @@ def test_key_value_set(self) -> None:
 
         assert mn.question == test_question
         assert mn.key == test_key
+        assert mn.key_to_log == f"'{test_key}'"
```

Comment:
I think this is the only test we need, as our typehints try to tell users to only pass `str`s to `MakeNote.as_`. If i add a test for passing a non-string to `as_`, MyPy complains about it. :P

This test shows that using `.as_` updates `key_to_log` at least.

Reply:
There isn't a test to prove `MakeNote(question, key)` displays the right thing, but I think if we're going to eventually make `key_to_log` into a property, we can address the problem there.